bootstrap-salt.sh

```diff
@@ -61,11 +61,11 @@ __DEFAULT_SLEEP=3
 _COLORS=${BS_COLORS:-$(tput colors 2>/dev/null || echo 0)}
 __detect_color_support() {
     if [ $? -eq 0 ] && [ "$_COLORS" -gt 2 ]; then
-        RC="\033[1;31m"
-        GC="\033[1;32m"
-        BC="\033[1;34m"
-        YC="\033[1;33m"
-        EC="\033[0m"
+        RC="\\033[1;31m"
+        GC="\\033[1;32m"
+        BC="\\033[1;34m"
+        YC="\\033[1;33m"
+        EC="\\033[0m"
```

Comment:
Since there are no variable references, we could just use single quotes here, right?